src/cmd/src/cli/export.rs

```diff
@@ -493,7 +493,7 @@ mod tests {
 
 ENGINE=mito
 WITH(
-  regions = 1
+  regions = '1'
```

Comment:
We anyway pass string underneath. For historical reason, we support pass option in number for `CreateTable`.

Reply:
Is this a breaking change?